tests/test_chiral_utils.py

```diff
@@ -229,6 +232,11 @@ def test_find_chiral_bonds():
 M  END"""
 
 
+def core_to_perm(core: NDArray, num_atoms_a: int) -> Sequence[int]:
+    a_to_b = {a: b for a, b in core}
+    return [a_to_b.get(a, UNMAPPED) for a in range(num_atoms_a)]
+
```

Comment:
location nit: utility fxn might be handy outside of chiral_utils_tests, maybe added to mcgregor.py or atom_mapping.py ?

Reply:
Moved in https://github.com/proteneer/timemachine/pull/1233/commits/88758e57be39a473f2409be60e243b084c20503a. Also factored out the inverse function `perm_to_core` and added a test.